packages/core-mobile/app/new/features/deposit/hooks/benqi/useBenqiAvailableMarkets.ts

```diff
@@ -171,6 +177,6 @@ export const useBenqiAvailableMarkets = (): {
     isPending: isPendingEnrichedMarkets,
     isFetching: isFetchingEnrichedMarkets,
     // eslint-disable-next-line @typescript-eslint/no-empty-function
-    refetch: cChainClient ? refetch : () => {}
+    refetch: networkClient ? refetch : () => {}
```

Comment:
you can also create a function (with useCallback) and do an early return when networkClient is invalid.